src/Raven.Server/ServerWide/Commands/Subscriptions/PutSubscriptionCommand.cs

```diff
@@ -26,6 +29,7 @@ public class PutSubscriptionCommand : UpdateValueForDatabaseCommand
         public bool Disabled;
         public string MentorNode;
         public bool PinToMentorNode;
+        public ArchivedDataProcessingBehavior ArchivedDataProcessingBehavior;
```

Comment:
this is not nullable, so old (5.4) commands will set default value here on deserialization? Is this what we want?

Reply:
wow, I didn't take that into account, that commands are stored as well
I'll think about it

Reply:
we'll leave it as nullable, then it'll be handled while reading